spec/kitchen/driver/openstack_spec.rb

```diff
@@ -1144,6 +1170,16 @@
     end
   end
 
+  describe '#wait_for_server' do
+    let(:state) { { hostname: 'hostname' } }
+
+    context 'standard WinRM sleep check' do
+      it 'calls the normal Kitchen WinRM wait' do
+        expect_any_instance_of(described_class).not_to receive(:sleep)
```

Comment:
Yeah it's strange because i just cargo culted a test to see if it would work. I'll remove it for now.